uvm/bootstrap/com/untangle/uvm/UvmContextSelector.java

```diff
@@ -0,0 +1,248 @@
+    private static final StatusLogger LOGGER = StatusLogger.getLogger();
+    public static final String UVM_LOG = "uvm";
+    private static final UvmContextSelector INSTANCE;
+    private static final ThreadLocal<String> THREAD_LOG_INFO = new InheritableThreadLocal<>();
+
+    static {
+        INSTANCE = new UvmContextSelector();
+    }
+
+    /**
+     * instance() Provides the UvmContextSelector singleton
+     * @return UvmContextSelector
+     */
+    public static UvmContextSelector instance() {
+        return INSTANCE;
+    }
+
+    /**
+     * Overrides getContext method of ContextSelector interface
+     * @param fqcn fqcn
+     * @param loader loader
+     * @param currentContext boolean isCurrentContext
+     * @return LoggerContext
+     */
+    @Override
+    public LoggerContext getContext(String fqcn, ClassLoader loader, boolean currentContext) {
+        try{
+            String contextName = THREAD_LOG_INFO.get();
+            synchronized (contexts) {
+                if (!contexts.containsKey(contextName)) {
+                    UvmLoggerContext context = new UvmLoggerContext(contextName);
+                    context.start();
+                    contexts.put(contextName, context);
+                }
+            }
+            return contexts.get(contextName);
+        } catch (Exception e) {
+            LOGGER.error("Exception: ", e);
+        }
+        return contexts.get(UVM_LOG);
+    }
+
+    /**
+     * Overrides getContext method of ContextSelector interface
+     * @param fqcn fqcn
+     * @param loader loader
+     * @param currentContext boolean isCurrentContext
+     * @param configLocation configLocation
+     * @return LoggerContext
+     */
+    @Override
+    public LoggerContext getContext(String fqcn, ClassLoader loader, boolean currentContext, URI configLocation) {
+        return getContext(fqcn, loader, currentContext);
+    }
+
+    /**
+     * Overrides removeContext method of ContextSelector interface
+     * @param context LoggerContext
+     */
+    @Override
+    public void removeContext(LoggerContext context) {
+        synchronized (contexts) {
+            LoggerContext remove = contexts.remove(context.getName());
+        }
+    }
+
+    /**
+     * Overrides getLoggerContexts method of ContextSelector interface
+     * @return List<LoggerContext>
+     */
+    @Override
+    public List<LoggerContext> getLoggerContexts() {
+        Collection<LoggerContext> values = contexts.values();
+        return new ArrayList<>(values);
+    }
+
+    /**
+     * Set the current thread's logging config to the "App" settings
+     * @param appId appId
+     */
+    public void setLoggingApp(Long appId) {
+        this.setThreadLoggingInformation("app-" + appId.toString());
+    }
+
+    /**
+     * Set the current thread's logging config to the "UVM" settings
+     */
+    public void setLoggingUvm() {
+        this.setThreadLoggingInformation("uvm");
```

Comment:
Use `UVM_LOG` constant instead of `uvm` string

Reply:
Changed the code to use UVM_LOG